roles/kubernetes/master/templates/kubeadm-config.yaml.j2

```diff
@@ -34,7 +34,12 @@ apiServerExtraArgs:
   bind-address: {{ kube_apiserver_bind_address }}
   insecure-bind-address: {{ kube_apiserver_insecure_bind_address }}
   insecure-port: "{{ kube_apiserver_insecure_port }}"
+{% if kube_version | version_compare('v1.10', '<') %}
   admission-control: {{ kube_apiserver_admission_control | join(',') }}
+{% else %}
+  enable-admission-plugins: {{ kube_apiserver_enable_admission_plugins | join(',') }} 
```

Comment:
Sorry to bother again but - I'm not sure of the behaviour in golang/k8s -  these parameters shouldn't even be defined if they're empty (i.e. when kube_apiserver_enable_admission_plugins kube_apiserver_disable_admission_plugins are empty lists).
It also applies to kube-apiserver.manifest.j2